helmfile.d/values/networkpolicies/common/crossplane.yaml.gotmpl

```diff
@@ -1,3 +1,10 @@
+{{- $functions := (list
+  "auto-ready"
+  "capability"
+  "go-templating"
+  "patch-and-transform"
+) }}
```

Comment:
I wonder if we could find a way to use the same list here and in the other place so that we just have to keep one list up to date instead of two?